packages/webapp/pages/tags/[tag].tsx

```diff
@@ -185,12 +192,34 @@ interface TagPageParams extends ParsedUrlQuery {
   tag: string;
 }
 
-export function getStaticProps({
+export async function getStaticProps({
   params,
-}: GetStaticPropsContext<TagPageParams>): GetStaticPropsResult<TagPageProps> {
+}: GetStaticPropsContext<TagPageParams>): Promise<
+  GetStaticPropsResult<TagPageProps>
+> {
+  let initialData: Keyword | null = null;
+
+  try {
+    const result = await request<{ keyword: Keyword }>(
+      graphqlUrl,
+      KEYWORD_QUERY,
+      {
+        value: params.tag,
+      },
+    );
+
+    if (result.keyword) {
+      initialData = result.keyword;
+    }
+  } catch (error) {
+    // keyword not found, ignoring for now
+  }
+
   return {
     props: {
       tag: params.tag,
+      initialData,
     },
+    revalidate: 3600,
```

Comment:
Since feed is client side fetched I think 1 hour is good time to cache this. We can potentially do even more.

Reply:
If we want we can also prerender some good keywords on each build with getStaticPaths.